frontend/src/components/Header.tsx

```diff
@@ -168,18 +167,18 @@ const Header = ({ locale }: Props) => {
         basic={true}
         className="desktop:position-sticky top-0 desktop:z-500 bg-white border-bottom-2px border-primary-vivid"
       >
-        <div className="usa-nav-container display-flex flex-align-end">
+        <div className="usa-nav-container display-flex flex-justify">
```

Comment:
This was the fix for the header alignment

Reply:
Oh, weird. This must've been a regression from the auth work @doug-s-nava (cc) did. Let's make sure the layout still works with the `authOn` feature flag enabled.

Reply:
not @doug-s-nava 
![ididit](https://github.com/user-attachments/assets/313ca1a0-bb37-41ff-9c86-049f7bd38c0a)

Reply:
<img width="608" alt="image" src="https://github.com/user-attachments/assets/7ffc1b2b-3445-40de-b260-f609e78f6174" />